src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -98,7 +102,7 @@ static ConnectionProvider create(String name) {
 		return builder(name).maxConnections(DEFAULT_POOL_MAX_CONNECTIONS)
 		                    .pendingAcquireMaxCount(500)
 		                    .pendingAcquireTimeout(Duration.ofMillis(DEFAULT_POOL_ACQUIRE_TIMEOUT))
-		                    .fifo();
+		                    .build();
```

Comment:
the builder has not yet been released, right?

Reply:
I see the `Builder` was introduced in #955, and that issue is in `0.9.5.RELEASE` milestone, but the commit is from Jan 16, and we released `0.9.4.RELEASE` on Jan 20 🤔

Reply:
yes 0.9.4 was release only with a fix that covers an incompatible change in 0.9.3
https://github.com/reactor/reactor-netty/compare/v0.9.3.RELEASE...v0.9.4.RELEASE